free-courses-en.md

```diff
@@ -161,6 +162,11 @@
 
 * [Functional Programming with Clojure](http://mooc.fi/courses/2014/clojure/)
 
+### Cloud computing and development
```

Comment:
Hello, your build is failing because there needs to be an extra space here. Check out [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
```suggestion
* [Functional Programming with Clojure](http://mooc.fi/courses/2014/clojure/)


### Cloud computing and development
```